app/assets/javascripts/admin/models/theme.js.es6

```diff
@@ -188,6 +188,12 @@ const Theme = RestModel.extend({
     const settings = {};
     settings[name] = value;
     return this.save({ settings });
+  },
+
+  saveTranslation(name, value) {
+    const translations = {};
+    translations[name] = value;
+    return this.save({ translations });
```

Comment:
I think we support the ES6 dynamic key syntax. If not we should enable that on babel! Would simplify this to:

`this.save({ translations: { [name]: value } });`